tests/data/test-data.sql

```diff
@@ -132,7 +132,7 @@ insert into leaderboard_rating (login_id, mean, deviation, total_games, leaderbo
   (102, 1500, 500, 0, 1),
   (102, 1500, 500, 0, 2),
   (105, 1400, 150, 20, 3),
-  (106, 1500, 75, 20, 3)
+  (106, 900, 75, 20, 3)
```

Comment:
Why is this change needed again?

Reply:
The new algorithm doesn't match newbies completely randomly. If the game is too unbalanced the players don't get matched at all. The integration test placed two 0 rated newbies against a 950 and a 1275 rated player. From the feedback I gathered the players don't find matches like these desireable. 
I lowered the rating to 950 and 675. This is still bad and at the edge of matching at all, but matches after three rounds when the maximum newbie time bonus is reached.
It passed before, because many really bad matchups would unintentionally get a quality rating of just below 0 instead of a way negative one.